### PR TITLE
HCAP-1233 | Return of service date in future

### PR DESCRIPTION
**Pull request by taraepp**
Ticket: https://freshworks.atlassian.net/browse/HCAP-1233

Testing things also added:
1. A cypress command to assign an array of sites to a user (should probably always be test-ha)
2. script to add table data from arbitrary tables. Currently they're hardcoded, but this could be expanded on in the future
3. Script relies on other pieces being run first: kc-import-users, and seed-participants- technically the sites and participants _could_ be included in this script, but the data sheets will need to be regenerated so that the columns in the spreadsheet match the columns in the table.

### CODE REVIEW

File: client/src/components/modal-forms/ReturnOfServiceForm.js [hunk 1]
```diff
@@ -127,7 +126,6 @@ export const ReturnOfServiceForm = ({
                 name='date'
                 component={RenderDateField}
                 label='Return of Service Start Date'
-                maxDate={getTodayDate()}
```
Comment (arranfw) [inline client/src/components/modal-forms/ReturnOfServiceForm.js:130]: Should maxDate be a value in line with 
> and before January 1st 2100

File: server/package.json [hunk 2]
```diff
@@ -13,6 +13,7 @@
     "migrate": "node-pg-migrate",
     "feed-participants": "node scripts/feed-participants.js",
     "feed-sites": "node scripts/feed-sites.js",
+    "feed-data": "node scripts/feed-data.js",
```
Comment (arranfw) [inline server/package.json:16]: Is this generic enough to be called `feed-data`? It looks like `post_secondary_institutions` is a hard value in the script

File: client/src/components/modal-forms/ReturnOfServiceForm.js (hunk 1 above)
Comment (taraepp) [reply client/src/components/modal-forms/ReturnOfServiceForm.js:131]: You are absolutely right!

File: cypress/support/commands.js [hunk 3]
```diff
@@ -200,3 +200,39 @@ Cypress.Commands.add('callAPI', ({ api, method = 'POST', body, followRedirect =
     });
   });
 });
+
+// assigns the site_ids to the username, assuming the region is already correct
+// does not un-assign previous sites
+Cypress.Commands.add('assignSitesToHealthAuthorityUser', (username, site_ids) => {
```
Comment (arranfw) [inline cypress/support/commands.js:206]: Awesome

**Review by arranfw — APPROVED** [review]
I didn't look too closely at the seeding script as we have plans to improve and expand it in the future, otherwise the rest of the work looks great, awesome job with `assignSitesToHealthAuthorityUser`

**Review by fw-noel — APPROVED** [review]

**Review by arranfw — APPROVED** [review]

**Pull request merged**